test/csuite/random_directio/main.c

```diff
@@ -134,20 +134,20 @@ static const char *const uri_rev = "table:rev";
  * create table:A101-3          (starts a new sequence)
  *
  * insert k/v 102 into table:main
- * rename table:A100-3 -> table:B100-3  (third step in sequence)
```

Comment:
It'd be good to chat offline about these changes. I don't know enough about what the `rename`s were trying to achieve to assess whether this is correct.